content/english/java-basics/activity-6.md

```diff
@@ -3,4 +3,182 @@ title: "Object and Classes"
+Usually, constructor is the method that initialializes value to all the fields in a class. It has the format `public` `class name` `(parameter)`. Since we have 5 fields in this class, the constructor will take in 5 parameter/inputs.
+```java
+public Bird (String species_input, String name_input, String hobby_input, int age_input, boolean loveMusic_input){
+    // constructor body
+}
+```
+In the constructor body, we need to initialize all the instance variables, by assign each variables to its initial values:
+```java
+species = species_input;
+name = name_input;
+hobby = hobby_input;
+age = age_input;
+loveMusic = loveMusic_input;
+```
+<br />
+
+#### Lastly, let's create some methods for the class `Bird`.
+We are gonna create 6 methods for this class!
+-  getSpecies();   // return species of the bird
+-  getName();      // return name of the bird
+-  getHobby();     // return hobby of the bird
+-  getAge();       // return age of the bird
+-  getLoveMusic(); // return whether the bird loves music
+-  toString();     // return information in a String
+
+Try to write out the first 5 methods with what you learned from the previous exercise about writing methods!
+
+<br />
+After, let's write the `toString()` method together!
+
+`toString()` is a method that returns the `String` representation of the object. 
+
+We will return a String that include all fields of the `Bird`: name, age, species, hobby, loveMusic by calling the 5 methods you just implemented!
+```java
+public String toString(){
+    // initialize variable info to have empty String
+    String info = "";
+
+    // append information to the String in the format: 
+    // Hi, my name is [name]. I am a [age] year old [species] who likes to [hobby].
+    info = info + "Hi, my name is " + getName() + 
+           ". I am a " + getAge() + " year old " + getSpecies() + 
+           " who likes to " + getHobby() + ". ";
+
+    // if the Bird likes music, append the String "I also like music very much!" to info
+    if(getLoveMusic()){
+        info = info + "I also like music very much!";
+    }else{        
+        info = info + "However, I do not like music!";
+    }
+
+    // return the full String
+    return info;
+}
+```
+{{% notice warning %}}
+If you correctly implemented the `Bird` class, you will see the following message when you click `Run`:
+```
+Congradulations! You correctly implemented the Bird Class :)
+```
+
+If there is still something incomplete or incorrect when you click `Run`:
+- You might get an error such as `Main.java:3: error: cannot access Bird`.
+- Or, you will see the message, `Something is still not quite right!`.
+{{% /notice %}}
+
+### 3. Creating Birds 🐥!
+Congradulations! You just wrote you first Java class `Bird` class. Now let's learn to write a program that uses `Bird` objects!
+
+We defined the `Bird` class to have the following attributes (instance variables) and bahaviors (methods):
+<img src="../images/bird_class.png" height="450"/> 
```

Comment:
Do we own the picture? if not, let's add credits 👍

Reply:
I did make the image for the classes!